diff --git a/parabem/__init__.py b/parabem/__init__.py
index ea0c80b..b690291 100755
--- a/parabem/__init__.py
+++ b/parabem/__init__.py
@@ -69,4 +69,4 @@ from parabem_cpp import vector3 as Vector3
 from parabem_cpp import PanelVector2, PanelVector3, Panel3, Panel2
 from parabem.utils import Vector
 
-from version import __version__
\ No newline at end of file
+from .version import __version__
\ No newline at end of file
